app/main/views/services.py

```diff
@@ -15,13 +15,26 @@
 
 from ..auth import role_required
 from ..helpers.diff_tools import get_diffs_from_service_data, get_revision_dates
+from ..helpers.service import approve_or_download_sentence_for_framework
 
 
 @main.route('', methods=['GET'])
 @login_required
 @role_required('admin', 'admin-ccs-category', 'admin-ccs-sourcing')
 def index():
-    return render_template("index.html")
+    try:
+        frameworks = data_api_client.find_frameworks()
```

Comment:
If there are no frameworks then something is very wrong - we probably shouldn't recover from that.